setup.py

```diff
@@ -32,7 +32,7 @@
     "docs": [
         "Sphinx>=1.4.4,<2.0",
         "sphinx-rtd-theme>=0.1.9",
-        "sphinx-click>=1.0.4",
+        "sphinx-click>=2.5.0",
```

Comment:
Could you at the same time amend the docstrings a bit so that the page looks prettier?  For example:

- [ ] `download-files` docstrings have two examples that are cutting lines; they could be shortened, or even simply removed, since the features like `–filter-range 1-2,3-4` are already showcased before

![2020-11-06-153257_691x380_scrot](https://user-images.githubusercontent.com/517546/98377640-5d97e500-2045-11eb-970a-3c4ac755f468.png)

Reply:
[ ] Please add also suitable starting "Usage" section, listing all the commands, as we do for `reana-client` https://reana-client.readthedocs.io/en/latest/#cli-api

Reply:
- [ ] docs for `get-metadata` also contains one example that is going for more than one line.  You can shorten it.